omf-python/src/file/reader.rs

```diff
@@ -108,6 +109,14 @@ impl PyReader {
             .map_err(|e| PyErr::new::<PyValueError, _>(e.to_string()))
     }
 
+    pub fn array_gradient(&self, array: &PyGradientArray) -> PyResult<Vec<[u8; 4]>> {
```

Comment:
Could add array method comments from https://gmggroup.github.io/omf-rust/rust/omf/file/struct.Reader.html#method.array_gradient

Reply:
What should we do about references to types in the docs do you think? e.g. `/// Read an [`array_type::Color`](crate::array_type::Color) array.`

Reply:
Hmm. I guess since the documentation has typing, they can already follow the array type to ColorArray, so maybe just 
`///Read a color array.`

is sufficient?

Reply:
Yeah sounds good, just as an aside, I'm trying to find the .pyi file after running `maturin develop` in the `omf-python` dir: 

```sh
find .. -type f -name "omf_python.pyi"
```

but it's not finding it anywhere, any tips on what I might be missing here?

Reply:
You need to run `cargo run --bin stub_gen` and that generates `omf_python.pyi`